build/scripts/install_tarball

```diff
@@ -17,7 +42,15 @@ else
     INSTALL_COMMAND="install -o ${PMM_USER} -g ${PMM_GROUP}"
 fi
 
-echo "Installing into ${PMM_DIR}..."
+# Check if PMM_DIR has the right permission to install files into it
+mkdir -p "${PMM_DIR}" || true
+if [ -w ${PMM_DIR} ]; then
+    echo "Installing into ${PMM_DIR}...";
+else
+    echo -e "${PMM_DIR} is not writable. \nPlease make sure the user $(id -un) has permissions to write to this directory\n\n"
```

Comment:
```suggestion
    echo -e "Cannot write to ${PMM_DIR}. \nPlease make sure the user $(id -un) has permissions to write to this directory\n\n"
```

Reply:
Quote from the source - https://www.gnu.org/software/bash/manual/bash.html

> -w file
True if file exists and is writable.

It seems like this is how the Linux community talks `bash` :)

Reply:
@ademidoff So, should we go with "writable" only?

Reply:
I agree with @nastena1606 's suggestion.